uniffi_bindgen/src/bindings/kotlin/templates/ObjectTemplate.kt

```diff
@@ -3,11 +3,19 @@
 {%- let (interface_name, impl_class_name) = obj|object_names %}
 {%- let methods = obj.methods() %}
 
-{% include "Interface.kt" %}
```

Comment:
I'm not sure, but I believe the interface definition enables delegation and some users are using this.  Can we keep it in?

Reply:
Ah, I didn't realise that was an option. We don't rely on delegation on our project as we try to destroy and map any Rust backed object as soon as possible, but it could be useful to add an intermediate wrapper, even though you'd still need to pass Rust backed objects to any functions that require them, although you could try to minimise these by tweaking the binding API.

I'll add back the interfaces then.